package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@entando/entando-bundle-cli",
-  "version": "1.0.0-SNAPSHOT",
+  "version": "1.1-SNAPSHOT",
```

Comment:
I would use `1.1.0-SNAPSHOT` and execute `npm run version` to regenerate the README file

Reply:
done